pkg/volume/azure_file/azure_provision.go

```diff
@@ -41,6 +41,8 @@ type azureCloudProvider interface {
 	CreateFileShare(name, storageAccount, storageType, location string, requestGB int) (string, string, error)
 	// delete a file share
 	DeleteFileShare(accountName, key, name string) error
+	// resize a file share
+	ResizeFileShare(accountName, accountKey, name string, sizeGiB int64) error
```

Comment:
ResizeFileShare uses `sizeGiB` here, CreateFileShare uses 'requestGB', that may be confusing. Since both use 'GiB' in fact, consider renaming `requestGB` to `sizeGiB`.